$NetBSD: patch-at,v 1.3 2016/07/03 19:18:47 dholland Exp $

- Honor the pkgsrc shell, to avoid getting a broken shell on e.g. Solaris.
- Don't use solaris make.
- Don't untar the runtime here since we did it from the pkgsrc makefile.

--- config/install.sh.orig	2011-05-16 21:13:32.000000000 +0000
+++ config/install.sh
@@ -55,7 +55,7 @@ cp config/preloads preloads.standard
 #
 MAKE=make
 
-SHELL=/bin/sh
+SHELL=@CONFIG_SHELL@
 vsay $this: Using shell $SHELL.
 
 #
@@ -291,7 +291,8 @@ esac
 
 case $OPSYS in
     solaris)
-	MAKE=/usr/ccs/bin/make
+	# not in pkgsrc!
+	#MAKE=/usr/ccs/bin/make
 	;;
     linux)
 	EXTRA_DEFS=`"$CONFIGDIR/chk-global-names.sh"`
@@ -335,7 +336,7 @@ BOOT_FILES=sml.$BOOT_ARCHIVE
 if [ -x "$RUNDIR"/run.$ARCH-$OPSYS ]; then
     vsay $this: Run-time system already exists.
 else
-    "$CONFIGDIR"/unpack "$ROOT" runtime
+#    "$CONFIGDIR"/unpack "$ROOT" runtime
     cd "$BASEDIR"/runtime/objs
     echo $this: Compiling the run-time system.
     $MAKE -f mk.$ARCH-$OPSYS $EXTRA_DEFS
